CHANGELOG.md

```diff
@@ -0,0 +1,293 @@
+
+Note that for technical reasons, we will be migrating our mobile client to a separate project with its own release notes in the future.
+
+#### Relevant Commits
+
+-   3f1f5c2b feat(mobile-alphabet): support configurable credits screen (#653)
+
+### Additional Infrastructure
+
+There is always some work that is opaque to users, but prerequisite to building out new features. Such is the case with the following work.
+
+#### Relevant Commits
+
+-   4e4cdd2d feat(api): dynamically configure allowed origins (#666)
+-   5b236b43 refactor(api): move media ingestion to the media service (#649)
+-   5cf931df refactor: isolate vertical slice for media item module (#648)
+-   426cc954 refactor: avoid high-level references to view collection names (#642)
+-   01676668 refactor(api): simplify event handlers (#625)
+-   7dd1d9f9 feat(api): publish events in command handlers (#623)
+-   a8e40ccf feat(data-types): append class name to nested data type metadata (#592)
+-   eab9c2ab feat(coscrad-frontend): add map view to spatial feature detail full view (#596)
+-   74bb6769 feat(api): introduce UNPUBLISH_RESOURCE (#600)
+-   380cc4bd Enhance song views 187564350 (#586)
+-   91bee25b migration(5): migrate events from legacy snapshot collections (#693)
+-   98c08356 infra(api): opt in to event sourced command repos (#680)
+
+### Web Alphabet App
+
+Our previous iteration of the interactive alphabet chart was a one-off project external to the core platform. We have now modernized this as part of the COSCRAD plaform. In the future, we will integrate the alphabet chart with the "search by letter" and "full text search" features, so that the alphabet can serve as an entry point into the web-of-knowledge.
+
+We have also ported this feature to the new mobile client.
+
+#### Relevant Commits
+
+-   6e1bb50d style(coscrad-frontend): style alphabet cards (#620)
+-   19db1b43 test(coscrad-frontend): add UX test for alphabet (#617)
+-   a961e7dd feat(coscrad-frontend): introduce alphabet chart (#614)
+-   d8ccac6d refactor(api): optimize media item queries by name (#613)
+-   bcc01f1a feat(api): define filename and filetype in binary downloads (#619)
+
+### Note enhancements
+
+Users can now add audio and translate notes. This makes it possible to create a web-of-knowledge where the notes and connections are in the language!
+
+#### Relevant Commits
+
+-   1e789c59 feat(coscrad-api): introduce ADD_AUDIO_FOR_NOTE (#589)
+-   dff3beed feat(api): introduce TRANSLATE_NOTE (#582)
+
+### Support for Additional Materials
+
+Sometimes users have materials that do not fit the mold for the available resource types. In this case, we support adding additional materials that can include audio, video, and PDF documents along with a description.
+
+We also now support a curratable list of links to external language resources on the web.
```

Comment:
Just wanted to make at least one comment, lol...

(sp): curatable

Reply:
I need to pipe this through a spell check.

Reply:
fixed